media/js/mozorg/contribute/contribute-2015.js

```diff
@@ -155,8 +155,17 @@
                 show_area(categoryarea);
             }
         } else {
+            // get and store the previous area before collapsing and hiding
+            // it's container else, we will not be able to get att it using
+            // the :visible psuedo selector.
+            var previouslySelectedArea = $('.area:visible');
+            previouslySelectedArea.find('select')
+                .prop('selectedIndex', 0)
+                .attr('required', false);
             $areascontainer.slideUp('fast', function() {
-                $('.area:visible').hide().find('select').prop('selectedIndex', 0);
+                // instead of using .hide() which is going to change in the
+                // upcoming jQuery 3 release, just remove the style attrbiute.
+                previouslySelectedArea.removeAttr('style');
```

Comment:
If I read the code correctly, it's the area container that is having `slideDown` applied. Here, you're removing the styles applied to `.area`, which is having `fadeIn` and `fadeOut` applied in the `show_area()` function.

But honestly, this is largely out of scope for this bug I think. I'm not sure if this will fix the issue for jQuery 3, but it still works here.